modules/pam_lastlog/pam_lastlog.c

```diff
@@ -665,9 +665,8 @@ last_login_failed(pam_handle_t *pamh, int announce, const char *user, time_t llt
 }
 
 /* --- authentication (locking out inactive users) functions --- */
-int
-pam_sm_authenticate(pam_handle_t *pamh, int flags,
-		    int argc, const char **argv)
+static int
+pam_auth(pam_handle_t *pamh, int flags, int argc, const char **argv)
```

Comment:
All other functions got the name of their respective module. Shouldn't this one be called `pam_lastlog` for consistency as well?

Reply:
Given that `pam_lastlog.so` provides non-trivial `pam_sm_open_session` and `pam_sm_close_session`,
a function called `pam_lastlog` would be somewhat confusing.

Reply:
Ah okay, makes sense then.